### DISCUSSION TITLE
What does the and symbol (`&`) do?

### DISCUSSION THREAD

**Discussion by PaulRBerg** [mermaid-js/mermaid#4385] [Q&A] [answered]
I saw the and symbol (`&`) used in a Mermaid.js diagram here:

https://github.com/ProjectOpenSea/seaport/blob/92841bcce9e1e49563f64585934f8040380ef1d2/README.md#diagram

But I'm not quite sure what its purpose is:

1. Does it horizontally align the nodes (this is what the ChatGPT said, though I'm skeptical this is the case)?
2. Is it a no-op syntax whose sole purpose is to improve the user experience by grouping similar nodes?

**Comment by elzup** [comment, marked as answer]
(`&`) should not have layout attributes.
https://mermaid.js.org/syntax/flowchart.html#chaining-of-links

it only shorthand for declare multiple.
https://github.com/mermaid-js/mermaid/blob/develop/packages/mermaid/src/diagrams/flowchart/parser/flow.jison#L374
